apps/server/src/models/groups.ts

```diff
@@ -99,21 +99,13 @@ Groups.init(
 )
 
 Groups.belongsTo(Users, {
-  foreignKey: 'adminId',
-  as: 'admin',
-  targetKey: 'id',
-})
-
-Users.hasMany(Groups, {
-  foreignKey: 'adminId',
-  as: 'admin',
+  foreignKey: 'userId',
+  as: 'user',
 })
 
-GroupWallet.belongsTo(Groups, {
-  foreignKey: 'groupId',
-})
-
-Groups.hasOne(GroupWallet, {
-  foreignKey: 'groupId',
+Groups.belongsTo(GroupWallet, {
+  foreignKey: 'userId',
+  as: 'groupWallet',
+  targetKey: 'id',
```

Comment:
This doesn't seem right. Following the DB design, GroupWallet has the id of the Group which is groupId, therefore, it means that GroupWallet belongs to Groups and not the other way round.